modules/scheduler/pre-existing-gke-cluster/versions.tf

```diff
@@ -20,11 +20,15 @@ terraform {
       source  = "hashicorp/google"
       version = "> 5.0"
     }
+    kubectl = {
+      source  = "gavinbunney/kubectl"
+      version = ">= 1.7.0"
+    }
   }
 
   provider_meta "google" {
     module_name = "blueprints/terraform/hpc-toolkit:pre-existing-gke-cluster/v1.37.2"
   }
 
-  required_version = ">= 1.0.0"
+  required_version = ">= 1.3"
```

Comment:
Can we remove this and update the root terraform required version ?

Reply:
Can you clarify this? I believe it is in the terraform root config.

Reply:
Ack